control/margins.py

```diff
@@ -522,10 +522,10 @@ def margin(*args):
         Gain margin
     pm : float
         Phase margin (in degrees)
-    wpc : float or array_like
-        Phase crossover frequency (where phase crosses -180 degrees)
     wgc : float or array_like
         Gain crossover frequency (where gain crosses 1)
+    wpc : float or array_like
+        Phase crossover frequency (where phase crosses -180 degrees)
```

Comment:
Are you sure?

If you compare with the [MATLAB function](https://www.mathworks.com/help/control/ref/lti.margin.html):  

```matlab
[Gm,Pm,Wcg,Wcp] = margin(sys)
```
`Wcg`, the  frequency corresponding to the gain margin, is the frequency, where the phase crosses 180°, and `Wcp`, the frequency for the phase margin, is the gain crossover frequency. We just have differently named variables but use the same order here and in `stability_margins`.

Reply:
I think you're right. I'll revert that part of this PR and adjust those names to correspond to MATLAB convention